molgenis-app/src/main/java/org/molgenis/app/promise/mapper/ParelMapper.java

```diff
@@ -280,14 +290,29 @@ private Entity toAgeType(String ageType)
 		List<String> unknown = Lists.newArrayList();
 		for (Entity sample : promiseSampleEntities)
 		{
-			String type = sample.getString("MATERIAL_TYPES");
-			if (materialTypesMap.containsKey(type))
+			String tissue = sample.getString("MATERIAL_TYPES_SUB");
```

Comment:
Ah, I understand. The field ```MATERIAL_TYPES_SUB``` was specifically added for this case, since the other cases already are distinguishable (like blood), and are just named in ```MATERIAL_TYPES```. Maybe the naming of the SUB field is a bit misleading, indeed. 

I'll flip the check around for readability and future proofing.